daemon/src/model.rs

```diff
@@ -637,30 +638,42 @@ impl Default for FundingRate {
 
 /// Fee paid on every contract renewal (a fraction of it if rolling over)
 #[derive(Debug, Clone, Copy, PartialEq, Serialize, Deserialize)]
-pub struct FundingFee(u64);
+pub struct FundingFee(i64);
 
 impl FundingFee {
     /// Value in satoshis
-    pub fn as_u64(&self) -> u64 {
+    pub fn as_i64(&self) -> i64 {
```

Comment:
Nit: I think the correct fn sig would be:

```suggestion
    pub fn to_i64(self) -> i64 {
```

because:

1. The type is copy, so consume `self`
2. The conversion is "expensive" because it involves more than pointers

Reply:
indeed! thanks for pointing it out!I followed bdk convention, I think they should have had `to_sats()` too!